minitests/roi_harness/basys3-x1y0-fixed.xdc

```diff
@@ -0,0 +1,44 @@
+# assign pins
+set_property -dict {PACKAGE_PIN V2 IOSTANDARD LVCMOS33} [get_ports {din[0]}]
+set_property -dict {PACKAGE_PIN W2 IOSTANDARD LVCMOS33} [get_ports {din[1]}]
+set_property -dict {PACKAGE_PIN V3 IOSTANDARD LVCMOS33} [get_ports {dout[0]}]
+set_property -dict {PACKAGE_PIN W3 IOSTANDARD LVCMOS33} [get_ports {dout[1]}]
+set_property -dict {PACKAGE_PIN W5 IOSTANDARD LVCMOS33} [get_ports {clk}]
+
+# create roi pblock
+create_pblock roi
+add_cells_to_pblock [get_pblocks roi] [get_cells -quiet [list roi]]
+resize_pblock [get_pblocks roi] -add {SLICE_X36Y0:SLICE_X65Y49}
+set_property CONTAIN_ROUTING 1 [get_pblocks roi]
+set_property EXCLUDE_PLACEMENT 1 [get_pblocks roi]
+
+# other stuff
+set_property DONT_TOUCH true [get_cells roi]
+set_property CFGBVS VCCO [current_design]
+set_property CONFIG_VOLTAGE 3.3 [current_design]
+
+# fix LUT locations
+
+## ins
+set_property BEL A6LUT [get_cells {roi/ins[0].lut}]
+set_property LOC SLICE_X64Y40 [get_cells {roi/ins[0].lut}]
+set_property BEL A6LUT [get_cells {roi/ins[1].lut}]
+set_property LOC SLICE_X64Y39 [get_cells {roi/ins[1].lut}]
+
+## outs
+set_property BEL A6LUT [get_cells {roi/outs[0].lut}]
+set_property LOC SLICE_X64Y38 [get_cells {roi/outs[0].lut}]
+set_property BEL A6LUT [get_cells {roi/outs[1].lut}]
+set_property LOC SLICE_X64Y37 [get_cells {roi/outs[1].lut}]
+
+# fix routes
+set_property FIXED_ROUTE { { IOB_IBUF0 RIOI_I0 RIOI_ILOGIC0_D IOI_ILOGIC0_O IOI_LOGIC_OUTS18_1 INT_INTERFACE_LOGIC_OUTS18 WL1BEG_N3 NW2BEG0 IMUX7 CLBLM_M_A1 }  } [get_nets {din_IBUF[0]}]
```

Comment:
Why is this required at all?  No IO is required from the ROI at all?

Reply:
So that I can test that I haven't broken anything by moving the ROI.